packages/framework-integration-tests/integration/end-to-end/cart.integration.ts

```diff
@@ -284,6 +285,9 @@ describe('Cart end-to-end tests', () => {
           `,
         })
 
+        console.log('Waiting 1 second for deletion to complete...')
+        await sleep(1000)
```

Comment:
I had to add this second because it seems like querying was "happening before deletion" or there was a non 100% sync flow. :wat: cc @javiertoledo @alvaroloes @NickSeagull

Reply:
Wouldn't this work with a `waitForIt`?

Reply:
hmm let me try, I didn't see that the delete doesn't have a waitForIt, that may be it

Reply:
nope it doesn't seem to work @NickSeagull

Reply:
Got it, I wouldn't spend much more time on this if it's working